### ISSUE TITLE
SQLite : Adding foreign key button is disabled.

### ISSUE THREAD

**Issue by o1lab** [xgenecloud/xgenecloud#20]
Reported by Stefanp from discord

**Comment by o1lab** [comment]
What appeared to be a UI issue turning out to be quite tricky. 

For SQLite, foreign keys can only be created while creating table. This is tricking up most GUI clients like Dbeaver/tableplus too! Solution would need an overhaul in GUI and lib on how  tableCreate with relationCreate are handled in databases.

**Comment by tluyben** [comment]
My ORM does it like it's detailed here https://stackoverflow.com/questions/1884818/how-do-i-add-a-foreign-key-to-an-existing-sqlite-table by the top answer. Not ideal, but there are not much other options.

**Comment by o1lab** [comment]
@tluyben : yeah, thank you. We are planning that way during table creation. What ORM are you using ?
